### PR TITLE
Correct some messages

### PR DESCRIPTION
**Pull request by pau-hedgehog**
Correct some usage messages and a log message

### CODE REVIEW

File: cmd/hhfab/main.go [hunk 1]
```diff
@@ -429,7 +429,7 @@ func Run(ctx context.Context) error {
 			},
 			{
 				Name:   "validate",
-				Usage:  "validate config and included wiring files",
+				Usage:  "validate config and include wiring files",
```
Comment (Frostman) [inline cmd/hhfab/main.go:432]: it validates config and validated included wiring files, should be reworded probably

File: pkg/fab/recipe/control_build.go [hunk 2]
```diff
@@ -176,7 +176,7 @@ func (b *ControlInstallBuilder) Build(ctx context.Context) error {
 		return fmt.Errorf("downloading cert-manager: %w", err)
 	}
 
-	slog.Info("Adding config and included wiring to installer", "control", b.Control.Name)
+	slog.Info("Adding config and including wiring to installer", "control", b.Control.Name)
```
Comment (Frostman) [inline pkg/fab/recipe/control_build.go:179]: ditto

**Review by Frostman — CHANGES_REQUESTED** [review]
thanks for picking it up!

File: cmd/hhfab/main.go (hunk 1 above)
Comment (pau-hedgehog) [reply cmd/hhfab/main.go]: Changed to `"validate config and wiring files"`

File: pkg/fab/recipe/control_build.go (hunk 2 above)
Comment (pau-hedgehog) [reply pkg/fab/recipe/control_build.go]: Changed to `Adding config and wiring files to installer`

File: cmd/hhfab/main.go [hunk 3]
```diff
@@ -505,6 +505,7 @@ func Run(ctx context.Context) error {
 			},
 			{
 				Name: "vlab",
+			        Usage: "operate Virtual Lab",
```
Comment (Frostman) [inline cmd/hhfab/main.go:508]: it's a wrong indent here (mix of tabs and spaces)

Comment (Frostman) [reply cmd/hhfab/main.go:508]: sorry, it wasn't caught by CI b/c of the https://github.com/githedgehog/fabricator/pull/238

File: cmd/hhfab/main.go [hunk 4]
```diff
@@ -505,6 +505,7 @@ func Run(ctx context.Context) error {
 			},
 			{
 				Name: "vlab",
```
[the remark this answers is not in the document]
Comment (pau-hedgehog) [reply cmd/hhfab/main.go]: Fixed. Thanks for the hint

**Review by Frostman — APPROVED** [review]

**Pull request merged**